libopenstorage/stork
# Added stork version annotation for backup, backupschedule and restore CR.

**What type of PR is this?**
> Uncomment only one and also add the corresponding label in the PR:
feature

**What this PR does / why we need it**:
Added stork version annotation, which will be updated by stork for backup, backupschedule and restore CR.

**Does this PR change a user-facing CRD or CLI?**:
no

**Is a release note needed?**:
N.A.

**Does this change need to be cherry-picked to a release branch?**:
2.5.1

**Manual Testing:
For backup object**:
```
[root@central-hail-hand-0 px-backup]# kubectl describe applicationbackup siva2-2fcc3c6 -n kube-system
Name:         siva2-2fcc3c6
Namespace:    kube-system
Labels:       <none>
Annotations:  portworx.io/backup-name: siva2
              portworx.io/backup-uid: 2fcc3c6b-c345-4e16-93a2-7064024f9bf5
              portworx.io/backuplocation-name: backuplocation1
              portworx.io/cluster-name: cluster1
              portworx.io/cluster-uid: 3f8342ae-d07c-43bf-89be-d5a95d74e3d1
              portworx.io/created-by: px-backup
              portworx.io/last-update: 2020-09-30T03:37:41.665120916Z
              portworx.io/org-id: org1
              portworx.io/skip-backup-location-name-check: true
              stork.libopenstorage.org/storkVersion: 2.5.0-eb6c180
API Version:  stork.libopenstorage.org/v1alpha1
Kind:         ApplicationBackup
...........
```
